### PR TITLE
Task 9

### PR DESCRIPTION
**Pull request by AnnaSovpel**

### CODE REVIEW

**Review by Killerkip10 — COMMENTED** [review]
## Homework 9
- (1.2) attribute is not use. (**-5** points); 
- (1.5) (**-20** points);

### Total 75 points

File: start_your_career/start_your_career.html [hunk 1]
```diff
@@ -6,64 +6,26 @@
...
-   </div>             
+<main class="picture">
+   <div class="career">
+     <div class="career_word"> Start your career</div>      
+   </div>   
 </main>
-   <section style="margin: 0px 20px;">
-    <h2 style="text-align:center;">
-               TRAININGS</h2>
+<footer>
```
Comment (Killerkip10) [inline start_your_career/start_your_career.html]: All your page is a big footer?)

File: start_your_career/start_your_career.html [hunk 2]
```diff
@@ -6,64 +6,26 @@
...
+   </div>
 </header>
-<main style="background-image: url(https://images.pexels.com/photos/583846/pexels-photo-583846.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=1700);
-             background-size: cover; background-repeat: no-repeat;">
-   <div style="min-height: calc(100vh - 52px); background-attachment: fixed; display: flex; align-items: center; justify-content: left;
-               color: white; font-size: 80px; width: 600px; margin-left: 220px; word-spacing: 10px;">
-                Start your career 
-   </div>             
+<main class="picture">
+   <div class="career">
```
Comment (Killerkip10) [inline start_your_career/start_your_career.html]: Redundant div

File: start_your_career/start_your_career.html [hunk 3]
```diff
@@ -85,70 +47,67 @@ <h2 style="text-align:center;">
...
+         <div class="block">Web design and computer graphics</div>
+         <div class="block">Marketing & Sales</div>
+         <div class="block">System and network administration</div>
+         <div class="clear"></div>
+    <h2>UPCOMING COURSES</h2>
     <hr>
-    <table style="border-collapse: collapse; border-radius: 5px; margin-top: 10px; margin-bottom: 10px; 
-                  width: 100%; background-color: white; table-layout: fixed;">
-   <tr style="border-bottom: 1px grey solid;">
+    <div class="table">
```
Comment (Killerkip10) [inline start_your_career/start_your_career.html:59]: class="table" and the again class="table"?)

File: start_your_career/style.css [hunk 4]
```diff
@@ -1,11 +1,80 @@
...
+    background-color: #eceeec; 
+    font-family: Roboto, sans-serif;  
+}
+.header {
+    background-color: #373937;
+    padding: 10px;
+}
+.main_href {
+    line-height: 28px; 
+    font-family: Roboto, sans-serif;
```
Comment (Killerkip10) [inline start_your_career/style.css]: You already set font-family in the body!

File: start_your_career/style.css [hunk 5]
```diff
@@ -1,11 +1,80 @@
...
+}
+.href {
+    color: white;
+    text-decoration: none;
+}
+.href_blog {
+    margin-right: 15px;
+    color: white;
```
Comment (Killerkip10) [inline start_your_career/style.css]: color: white;
 text-decoration: none;

Already exist in the .href

File: start_your_career/style.css [hunk 6]
```diff
@@ -1,11 +1,80 @@
...
+    padding-right: 300px;
+}
+.section {
+    margin: 0px 20px;
+}
+ h2 {
     text-align:center; 
     margin: 25px;
-    font-family: Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    font-family: Roboto, sans-serif;
```
Comment (Killerkip10) [inline start_your_career/style.css]: Also redundant

File: start_your_career/style.css [hunk 7]
```diff
@@ -51,6 +121,74 @@
...
+    }
+}
+@media (max-width: 1024px){
+    .table {
+        table-layout: auto;
+        overflow: auto;
+        max-width: 100%;
+    }
+}
+@media (min-width: 1024px){
```
Comment (Killerkip10) [inline start_your_career/style.css:166]: Please sort your @media by max-width 1024, 750, 375

File: start_your_career/start_your_career.html (hunk 1 above)
Comment (AnnaSovpel) [reply start_your_career/start_your_career.html]: Done.

File: start_your_career/start_your_career.html (hunk 2 above)
Comment (AnnaSovpel) [reply start_your_career/start_your_career.html]: Done.

File: start_your_career/start_your_career.html (hunk 3 above)
Comment (AnnaSovpel) [reply start_your_career/start_your_career.html:59]: The first div with class "table" is necessary for the adaptive table.

File: start_your_career/style.css (hunk 4 above)
Comment (AnnaSovpel) [reply start_your_career/style.css]: Done.

File: start_your_career/style.css (hunk 5 above)
Comment (AnnaSovpel) [reply start_your_career/style.css]: Done.

File: start_your_career/style.css (hunk 6 above)
Comment (AnnaSovpel) [reply start_your_career/style.css]: Done.

File: start_your_career/style.css (hunk 7 above)
Comment (AnnaSovpel) [reply start_your_career/style.css:166]: Done.

File: start_your_career/start_your_career.html [hunk 8]
```diff
@@ -85,70 +44,68 @@ <h2 style="text-align:center;">
       Vel incidunt dolorum obcaecati, hic rem quaerat quasi quo officia esse voluptate, autem, nostrum aliquid accusantium? Tempora veritatis impedit, vel tenetur sapiente repudiandae et culpa facilis? Repellat, aspernatur natus! Dicta.
       Odio commodi eum repellendus dolores velit delectus, esse voluptatibus quam impedit magni quisquam maiores quidem voluptatem minima. At cupiditate sit animi asperiores, eum ipsum enim doloribus provident quam corporis mollitia!
     </div>
-
-    <h2 class="headers">DEMANDED PROGRAMS</h2>
+    <footer>
```
Comment (Killerkip10) [inline start_your_career/start_your_career.html]: Again)
Half page is a footer?)

Comment (AnnaSovpel) [reply start_your_career/start_your_career.html]: Done.

**Pull request merged**